Integrations/src/test/java/io/deephaven/integrations/learn/ScattererTest.java

```diff
@@ -0,0 +1,19 @@
+package io.deephaven.integrations.learn;
+
+import org.junit.Test;
+
+import java.util.function.Function;
+
+public class ScattererTest {
+
+    @Test
+    public void createScattererTest() {
+        final Function<Object[], Object> func = args -> args;
+        Output[] outputs = new Output[]{new Output("OutCol1", func, "int"),
+                                        new Output("OutCol2", func, "boolean"),
+                                        new Output("OutCol3", func, "long")};
+
+        Scatterer scatterer = new Scatterer(outputs);
+    }
+
```

Comment:
I assume you want some more testing.

Reply:
You should run code coverage in IJ to see what is and is not being tested.  Use that to direct what unit tests need to be written.